website/content/en/docs/concepts/scheduling.md

```diff
@@ -602,4 +602,6 @@ topologySpreadConstraints:
 - maxSkew: 1
   topologyKey: capacity-spread
   whenUnsatisfiable: DoNotSchedule
+  labelSelector:
```

Comment:
This needs to be added to `v0.32` docs as well

Reply:
Should it be backported to all the current docs or just v0.32? It applies to all versions but not sure what the standard practice for docs is.